.env.dist

```diff
@@ -5,7 +5,7 @@ PUBLIC_PORT_GATEWAY=8081
 PUBLIC_PORT_PATTERN_LIBRARY=8082
 PUBLIC_PORT_DUMMY_API=8083
 # TODO: put this to latest as a default, when `latest` is available
-REVISION_BROWSER=dc3a02d8a8c9a75bac19e1a99d55d39b591d94fb
+REVISION_BROWSER=4a530a47f8d9d06759b81145bbd949482d775c8e
```

Comment:
https://github.com/libero/scripts/ now has pushing of `latest`, if its version is updated in `browser`

Reply:
Nope, let's follow up with that.